### PR TITLE
OCPBUGS-23125: add access review for impersonate

### PR DESCRIPTION
**Pull request by rhamilto**
Thanks to @spadgett for the assist on this one.

### CODE REVIEW

File: frontend/public/components/RBAC/bindings.jsx [hunk 1]
```diff
@@ -146,6 +146,14 @@ const menuActions = ({ subjectIndex, subjects }, startImpersonate) => {
     actions.unshift(() => ({
       label: i18next.t('public~Impersonate {{kind}} "{{name}}"', subject),
       callback: () => startImpersonate(subject.kind, subject.name),
+      // Must use API group authorization.k8s.io, NOT user.openshift.io
+      // See https://kubernetes.io/docs/reference/access-authn-authz/authentication/#user-impersonation
+      accessReview: {
+        group: 'authorization.k8s.io',
+        resource: subject.kind === 'Group' ? 'groups' : 'users',
+        name,
```
Comment (spadgett) [inline frontend/public/components/RBAC/bindings.jsx:154]: Is `name` here `subject.name` or a different value? It should match `subject.name`.

Comment (rhamilto) [reply frontend/public/components/RBAC/bindings.jsx:154]: Good catch.  Let me fix.

**Comment by rhamilto** [comment]
/label acknowledge-critical-fixes-only

typing is hard

File: frontend/public/components/RBAC/bindings.jsx (hunk 1 above)
Comment (spadgett) [reply frontend/public/components/RBAC/bindings.jsx]: Sorry, missed this. Fix looks good to me. Thanks!

**Pull request merged**